src/render/draw_custom.js

```diff
@@ -28,13 +33,38 @@ function drawCustom(painter: Painter, sourceCache: SourceCache, layer: CustomSty
             context.setColorMode(painter.colorModeForRenderPass());
 
             prerender.call(implementation, context.gl, painter.transform.customLayerMatrix());
+            if (painter.transform.projection.name === "globe") {
+                prerender.call(implementation, context.gl, painter.transform.customLayerMatrix(), painter.transform.getProjection(), painter.transform.globeToMercatorMatrix(),  globeToMercatorTransition(painter.transform.zoom));
+            } else {
+                prerender.call(implementation, context.gl, painter.transform.customLayerMatrix());
```

Comment:
If I'm reading this right, for non-globe projections, this line will be called twice... A mistake probably?

Reply:
Yes, this one would be a regression. We need to address this before the release.